panic-semihosting/Cargo.toml

```diff
@@ -20,3 +20,6 @@ cortex-m-semihosting = { path = "../cortex-m-semihosting", version = ">= 0.3, <
 exit = []
 inline-asm = ["cortex-m-semihosting/inline-asm", "cortex-m/inline-asm"]
 jlink-quirks = ["cortex-m-semihosting/jlink-quirks"]
+
+[package.emtadata.docs.rs]
```

Comment:
Aw, nuts. https://docs.rs/panic-semihosting/0.5.5/panic_semihosting/

Reply:
Aw, sorry for missing that